packages/api/src/auth/verifiers/__tests__/jwtVerifier.test.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  createVerifier,
+  WebhookSignError,
+  WebhookVerificationError,
+} from '../index'
+
+const payload = { data: 'No more secrets, Marty.' }
+const secret = 'MY_VOICE_IS_MY_PASSPORT_VERIFY_ME'
+
+describe('jwtVerifier verifier', () => {
+  describe('signs a payload', () => {
+    test('it has a signature', () => {
+      const { sign } = createVerifier({
+        options: { type: 'jwtVerifier' },
```

Comment:
What do you think of moving type to the top level so it looks like this?

```js
createVerifier('jwtVerifier', {...options})
```

Reply:
My reasoning - the first argument is for the "verifier factory" aka "verifier creator". The second options is for the actual verifier